### DISCUSSION TITLE
Destination Phone Number as Output

### DISCUSSION THREAD

**Discussion by FabrizioMoggio** [camaraproject/CallForwardingSignal#15] [Ideas]
According to action item 1-4: https://wiki.camaraproject.org/display/CAM/2024-03-05+CFS+Minutes

Discussion on phone number, target of the call FWD, as output of the API.

**Comment by FabrizioMoggio** [comment]
**Proposal**: don't have it as output for privacy reasons. 

The currently identified use cases, UseCase-1 and UseCase-2 (https://github.com/camaraproject/CallForwardingSignal/discussions/3) don't require it.

The currently identified Intent, Intent-1, (https://github.com/camaraproject/CallForwardingSignal/discussions/6), doesn't require it.

**Reply by chinaunicomyangfan** [reply]
Based on use case 2, an extended use case can be created. If the police can obtain the destination number of the call, they can combine interfaces such as user location to quickly locate the address of the fraudster, thereby improving the efficiency of arrest. However, due to privacy reasons, the destination number may only be provided to police or government departments. Further discussion can be conducted to provide APIs with different privacy levels for different target users

**Reply by FabrizioMoggio** [reply]
The API provides an indication of a POTENTIAL fraud. For this reason it would be better not to return the Destination Number to everyone. So policies are required, but also in case of an authorized API Consumer (e.g. the police) the Destination Phone number should be provided only in specific circumstances, only if a fraud act is confirmed. Don't you think? Maybe we should check with a legal office. Anyway, technically, these policies could be not easy to handle.

**Reply by chinaunicomyangfan** [reply]
I agree, not putting the number in the output is a good choice

**Comment by FabrizioMoggio** [comment]
**Conclusion:**  Destination Phone Number is not an output for the API 

According to the currently identified use cases (1, 2 and 3)

**Comment by FabrizioMoggio** [comment]
Closing according to: https://wiki.camaraproject.org/display/CAM/2024-MM-DD+CFS+Minutes
